web/oauth.go

```diff
@@ -135,10 +135,16 @@ func authorizeOAuthPage(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	// here we should check if the user is logged in
 	if c.App.Session().UserId == "" {
+		requestURI := r.RequestURI
+		subpath, err := utils.GetSubpathFromConfig(c.App.Config())
```

Comment:
Incase of parsing error, it will continue like earlier. I was in conflict whether to handle the error or not as this check is made additionally.

Reply:
I guess the implication of that is that if, for some reason, the config is `nil` or the `ServiceSettings.SiteURL` is invalid then this will use the `(http.Request).RequestURI`. That makes sense not to fail the login because the instance may not need a `SiteURL` value; it may just be some other bug. I'm wondering if it's worth logging it as an error or warning? I hate to think of OAuth requests failing *because of* a misconfigured `SiteURL` and that being hard to diagnose.

Reply:
Logging the parse error as warning and will continue the flow.